.travis_BASE_2549.yml

```diff
@@ -0,0 +1,33 @@
+sudo: required
```

Comment:
Looks like a local copy of the actual file that can be removed.

Reply:
I didn't understand how to deal with this one. I'll try to remove the local files and use `git pull` again.

Reply:
Done.